### ISSUE TITLE
Read

### ISSUE THREAD

**Issue by jokva** [Statoil/segyio#337]

**Comment by mrava87** [comment]
When it comes to examples, in my opinion the README file is not the best place. I would suggest taking a look into sphinx-gallery and add some examples with testdata to the readthedocs documentation if possible :)

**Comment by jokva** [comment]
That's a good suggestion - we've been wanting to beef up the readthedocs page with a better examples section.

I still think there's a place for some motivating examples in the readme too, though, at least to get people going.

**Comment by mrava87** [comment]
Agree. They are not mutually exclusive :) having something in README to give people a feeling of what the library can do is important, I was suggesting Sphinx-gallery if you want something a bit more advanced with prints and/or plots which I think would help people find out different features that maybe are too advanced for the README file :)

**Comment by jokva** [comment]
I think that's an excellent idea.

**Comment by megies** [comment]
Love is all you need <3
